src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingUtils.kt

```diff
@@ -364,33 +366,29 @@ fun Vilkårsvurdering.kopierResultaterFraForrigeBehandling(
             if (personResultatForrigeBehandling == null) {
                 initieltPersonResultat.vilkårResultater
             } else {
-                initieltPersonResultat.vilkårResultater
+                initieltPersonResultat
                     .overskrivMedVilkårResultaterFraForrigeBehandling(
-                        kunForGodkjenteVilkår = behandling.type != BehandlingType.FØRSTEGANGSBEHANDLING,
                         vilkårResultaterFraForrigeBehandling = personResultatForrigeBehandling.vilkårResultater,
-                        nyttPersonResultat = initieltPersonResultat,
                     )
             }
 
         initieltPersonResultat.setSortedVilkårResultater(oppdaterteVilkårResultater.toSet())
     }
 }
 
-private fun Collection<VilkårResultat>.overskrivMedVilkårResultaterFraForrigeBehandling(
+private fun PersonResultat.overskrivMedVilkårResultaterFraForrigeBehandling(
     vilkårResultaterFraForrigeBehandling: Collection<VilkårResultat>,
-    nyttPersonResultat: PersonResultat,
-    kunForGodkjenteVilkår: Boolean,
 ): List<VilkårResultat> {
-    val vilkårForPerson = nyttPersonResultat.vilkårResultater.map { it.vilkårType }.toSet()
+    val vilkårForPerson = this.vilkårResultater.map { it.vilkårType }.toSet()
 
     return vilkårForPerson.flatMap { vilkårType ->
         val vilkårResultaterAvSammeType: List<VilkårResultat> =
-            nyttPersonResultat.vilkårResultater.filter { it.vilkårType == vilkårType }
+            this.vilkårResultater.filter { it.vilkårType == vilkårType }
 
         val vilkårResultaterAvSammeTypeIForrigeBehandling =
             vilkårResultaterFraForrigeBehandling
                 .filter { it.vilkårType == vilkårType }
-                .map { it.kopier(personResultat = nyttPersonResultat) }
+                .map { it.kopier(personResultat = this) }
 
         val vilkårResultaterForrigeBehandlingSomViØnskerÅTaMed: List<VilkårResultat> =
             when (vilkårType) {
```

Comment:
Her har jeg gjort en refaktorering, siden intellij klaget på at receiving parameter aldri ble brukt. Endrer det til å være PersonResultat i stedet slik at nyttPersonResultat kan erstattes med this.